lib/experimental/media/class-gutenberg-rest-attachments-controller.php

```diff
@@ -0,0 +1,350 @@
+<?php
+/**
+ * Class Gutenberg_REST_Attachments_Controller.
+ *
+ * @package MediaExperiments
+ */
+
+/**
+ * Class Gutenberg_REST_Attachments_Controller.
+ */
+class Gutenberg_REST_Attachments_Controller extends WP_REST_Attachments_Controller {
+	/**
+	 * Registers the routes for attachments.
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes(): void {
+		parent::register_routes();
+
+		$args                       = $this->get_endpoint_args_for_item_schema();
+		$args['generate_sub_sizes'] = array(
+			'type'        => 'boolean',
+			'default'     => true,
+			'description' => __( 'Whether to generate image sub sizes.', 'gutenberg' ),
+		);
+		$args['convert_format']     = array(
+			'type'        => 'boolean',
+			'default'     => true,
+			'description' => __( 'Whether to support server-side format conversion.', 'gutenberg' ),
+		);
```

Comment:
I think either the argument name or the description needs an adjustment. If I understand correctly, you're saying "Whether to _support_ ..." because WP core doesn't do any of this out of the box right and it's left for plugins? If so, I would rename the argument to something like `support_format_conversion`.

Alternatively, we just focus on the fact that the server-side currently _can_ convert formats (whether via core or plugins), in that case `convert_format` is a good name, but then I think the description should also be more active and less about supporting. For example like the above:
```suggestion
		$args['convert_format']     = array(
			'type'        => 'boolean',
			'default'     => true,
			'description' => __( 'Whether to convert image formats.', 'gutenberg' ),
		);
```